zebra-state/Cargo.toml

```diff
@@ -20,3 +20,8 @@ color-eyre = "0.3.4"
 eyre = "0.4.2"
 tokio = { version = "0.2.21", features = ["full"] }
 zebra-test-vectors = { path = "../zebra-test-vectors/" }
+spandoc = { git = "https://github.com/yaahc/spandoc.git" }
+tracing = "0.1.15"
+tracing-futures = "0.2.4"
+tracing-error = "0.1.2"
+tracing-subscriber = "0.2.5"
```

Comment:
boop boop all aboard the tracing train

Reply:
this is actually partially a bug in spandoc, its supposed to bring in the tracing and tracing-futures deps but it doesn't for some reason, need to figure out how to fix that but for now its fine to just bring them in separately/